advanced_animation/audio/audio_generator.py

```
@@ -244,6 +281,132 @@ def get_supported_languages(self) -> Dict[str, str]:
+            
+            # Mix narration and background
+            mixed = narration.overlay(background)
+            
+            # Save the result
+            mixed.export(audio_path, format="mp3")
+            logger.info(f"Added background music to: {audio_path}")
+            
+        except Exception as e:
+            logger.error(f"Error adding background music: {e}")
+    
+    def generate_subtitles(self, text: str, output_path: str, language: str = 'en') -> bool:
```

Ruff (0.14.13)

Unused method argument: `language`

(ARG002)